public/index.html

```diff
@@ -151,7 +170,11 @@
     <header>
       <h1 class="visually-hidden">Excalidraw</h1>
     </header>
-    <div id="root"></div>
+    <div id="root">
+      <div class="LoadingMessage">
+        <span>Loading scene...</span>
+      </div>
+    </div>
```

Comment:
This hinges on the fact that React's initial render removes non-component children. But React docs are weird:

> ReactDOM.render() does not modify the container node (only modifies the children of the container). It may be possible to insert a component to an existing DOM node without overwriting the existing children.

So not sure here...

Reply:
Could you have this outside of the root, then remove it `componentDidMount` on `<App />`?

Reply:
It appears that `componentDidMount` is called after initial render so we'd need to use different className so as not remove the message inserted by React. Let's keep it as is and do that when we encounter issues.